multires_consensus_clustering/uncertainty_measure.py

```diff
@@ -0,0 +1,48 @@
+import igraph as ig
+import numpy as np
+import scanpy as sc
+import pandas as pd
+import multires_consensus_clustering as mcc
+import matplotlib.pyplot as plt
+
+
+def uncertainty_measure_cells(graph, adata, plot):
```

Comment:
I would make the plot a different function rather than an argument to this one (or just use `sc.pl.umap()` as needed).